src/include/function/unary_function_executor.h

```diff
@@ -22,6 +22,17 @@ struct UnaryFunctionWrapper {
     }
 };
 
+struct UnaryPointerFunctionWrapper {
```

Comment:
Call this UnarySequenceFunctionWrapper for now. Also open an issue about adding comments & unify Unary/BinaryFunctionWrapper and assign it to @acquamarin